### ISSUE TITLE
Unable to obtain a well-trained perception module with the DDP script

### ISSUE THREAD

**Issue by hollow-503** [CollaborativePerception/V2Xverse#24]
Thanks for your impressive work!
I try reproducing the results presented in the paper. Due to the limited disk space, I only generate a subset of data with the given script, which comprising:
![Snipaste_2024-08-12_17-12-19](https://github.com/user-attachments/assets/063611dd-1e0b-4b5f-ae9e-dae4a5280a52)
To reproduce the results, I firstly trained the perception module with the command:
```
CUDA_VISIBLE_DEVICES=0,1,2,3 python -m torch.distributed.launch  --nproc_per_node=4 --use_env opencood/tools/train_ddp.py -y opencood/hypes_yaml/v2xverse/codriving_multiclass_config.yaml
```
Following your last commit, I uncomment the `line 189` in [opencood/tools/train_ddp.py](https://github.com/CollaborativePerception/V2Xverse/commit/64e3d6369b9f26c461fa8d03a427a77f8c8777b1#diff-b6be42a9ade393a7fa56d21d37f8d71eba570b256cb844ac984466c73184e89e). However, the validation loss was still high at the epoch 3:
![Snipaste_2024-08-12_17-27-36](https://github.com/user-attachments/assets/5bcbb828-a4ec-4369-a34b-d8121b2d8fe0)
Is it correct? Should I continue training? 
Meanwhile, I evaluate the checkpoint of epoch 3 and get the following visulization results:
![bev_00160](https://github.com/user-attachments/assets/f1c1a4db-d6bf-404e-b28d-d88829d496b4)
![bev_00320](https://github.com/user-attachments/assets/83483e52-8341-4ee2-8b92-7ca09c29a588)
It seems that the perception module was hardly trained. Could you kindly give me some advice to reproduce the result shown in the paper?

**Comment by melihyazgan** [comment]
Hey did you figure it out?
